felles/oidc/src/main/java/no/nav/vedtak/sikkerhet/oidc/token/impl/TokenXExchangeKlient.java

```diff
@@ -72,22 +64,8 @@ private static HttpRequest.BodyPublisher ofFormData(OpenIDToken token, String as
         return HttpRequest.BodyPublishers.ofString(formdata, UTF_8);
     }
 
-    private String audience(URI uri) {
-        String host = uri.getHost();
-        var elems = host.split("\\.");
-        var joiner = new StringJoiner(":");
-        joiner.add(cluster);
-
-        if (elems.length == 1) {
-            joiner.add(namespace);
-            joiner.add(elems[0]);
-            return joiner.toString();
-        }
-        if (elems.length == 2) {
-            joiner.add(elems[1]);
-            joiner.add(elems[0]);
-            return joiner.toString();
-        }
-        throw new IllegalArgumentException("Kan ikke analysere " + host + "(" + elems.length + ")");
+    private static String audience(String scope) {
+        //Støtter bare ett scope
+        return scope.replaceFirst("api://", "").replace("/.default", "");
```

Comment:
Sikker på dette? https://docs.nais.io/security/auth/tokenx/?h=tokenx 
Se under assertion. TokenX bruker vel ikke scope fra restclientconfig

aud	https://tokendings.dev-gcp.nais.io/token	The audience of the token. Must be set to the token_endpoint of [Tokendings](https://github.com/nais/tokendings). The value of this exists in the metadata found at the [well-known endpoint]